File: app/src/main/java/com/lamnt/foodorder/view/fragment/payment/PaymentFragment.java
package com.lamnt.foodorder.view.fragment.payment;

import android.view.View;

import com.lamnt.foodorder.R;
import com.lamnt.foodorder.view.fragment.base.BaseFragment;

public class PaymentFragment extends BaseFragment {

    public PaymentFragment() {
    }


    @Override
    public int getLayoutRes() {
        return R.layout.fragment_payment;
    }

    @Override
    protected void unit(View v) {

    }

    @Override
    protected boolean isShowNotificationIcon() {
        return false;
    }

    @Override
    protected boolean isShowSearchIcon() {
        return false;
    }

    @Override
    protected boolean isShowBottomNav() {
        return false;
    }

    @Override
    public void onResume() {
        super.onResume();
        setTitleActionBar(getString(R.string.text_choose_payment_options));
    }
}

File: app/src/main/java/com/lamnt/foodorder/view/staff/fragment/payment/PaymentFragment.kt
package com.lamnt.foodorder.view.staff.fragment.payment

import com.lamnt.foodorder.R
import com.lamnt.foodorder.databinding.FragmentPaymentBinding
import com.lamnt.foodorder.view.base.BaseFragmentMVVM
import com.lamnt.foodorder.viewmodel.PaymentFragmentViewModel

class PaymentFragment : BaseFragmentMVVM<PaymentFragmentViewModel,FragmentPaymentBinding>() {
    override fun getLayoutRes(): Int {
        return R.layout.fragment_payment
    }

    override fun setTitle(): Int {
       return R.string.text_choose_payment_options
    }

    override fun initViewModel() {

    }

    override fun initObserver() {

    }

    override fun initViews() {

    }

    override fun initDataBinding() {

    }

    override fun initData() {

    }

}